joomla/joomla-cms
# [5.2] Remove alt-text for menuItem with image and title

Pull Request for Issue # .

### Summary of Changes
In the menu layouts menus can have an image followed by an text. Alt-text for images are generated equal to the menuitem title.  If the menutype is a link, this is an a11y issue, in other cases it is useless. 

This PR replaces the alt text by an empty alt-text for all menutypes.


### Testing Instructions
Code inspect and sourcecode inspect. 
Or use a tool like IBM accessible audit in your browser and check your menus with images. 



### Actual result BEFORE applying this Pull Request

![grafik](https://github.com/joomla/joomla-cms/assets/1035262/66f08b95-f01d-4a9b-9a39-c075b0caa24b)



### Expected result AFTER applying this Pull Request

![grafik](https://github.com/joomla/joomla-cms/assets/1035262/7f21835c-a7a0-4549-b571-657a8013d8e3)



### Link to documentations
Please select:
- x] No documentation changes for docs.joomla.org needed

- [x No documentation changes for manual.joomla.org needed
